src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.CompletionListUpdater.cs

```diff
@@ -818,18 +819,18 @@ private ItemSelection GetDefaultsMatch(IReadOnlyList<MatchResult> items, ItemSel
                     // so we just need to search for the first item that matches the suggested default.
                     for (var i = 0; i < inferiorItemIndex; ++i)
                     {
-                        if (items[i].CompletionItem.DisplayText == defaultText)
+                        if (items[i].CompletionItem.FilterText == defaultText)
```

Comment:
I assume this is the brunt of the change?

Reply:
Yes, this is the part that makes sure we'd select the default item if it is also a starred item:
1. Starred item is a duplicate of an existing item, for example, if we have item `★ ToString`, we'd also have a `ToString` in the list.
2. The DisplayText of starred item is prefixed with `★`, but its FilterText doesn't.

Therefore, comparing default with `DisplayText` would choose non-starred item instead.

That said, the most accurate approach would be to strip the `★` from DisplayText and then compare, but I do have a few other changes around starred item in mind and would like to get to that first. For now, FilterText would do :)